auth/lib/google_oauth2_example/endpoint.ex

```diff
@@ -0,0 +1,40 @@
+defmodule GoogleOAuth2Example.Endpoint do
+  use Phoenix.Endpoint, otp_app: :google_oauth2_example
+
+  socket "/socket", GoogleOAuth2Example.UserSocket
+
+  # Serve at "/" the static files from "priv/static" directory.
+  #
+  # You should set gzip to true if you are running phoenix.digest
+  # when deploying your static files in production.
+  plug Plug.Static,
+    at: "/", from: :google_oauth2_example, gzip: false,
+    only: ~w(css fonts images js favicon.ico robots.txt)
+
+  # Code reloading can be explicitly enabled under the
+  # :code_reloader configuration of your endpoint.
+  if code_reloading? do
+    socket "/phoenix/live_reload/socket", Phoenix.LiveReloader.Socket
+    plug Phoenix.LiveReloader
+    plug Phoenix.CodeReloader
+  end
+
+  plug Plug.RequestId
+  plug Plug.Logger
+
+  plug Plug.Parsers,
+    parsers: [:urlencoded, :multipart, :json],
+    pass: ["*/*"],
+    json_decoder: Poison
+
+  plug Plug.MethodOverride
+  plug Plug.Head
+
+  plug Plug.Session,
+    store: :cookie,
+    key: "_google_oauth2_example_key",
+    signing_salt: "TN3lMCf5"
```

Comment:
This should probably be in `prod.secret.exs`

Reply:
Do we feel strongly enough about this to block on merge?

Reply:
At least we should make this a dummy value or add a comment that this should go in a secret.

Reply:
Done!